app/webpacker/css/admin/connected_apps.scss

```diff
@@ -13,3 +13,13 @@
     margin-bottom: 1em;
   }
 }
+
+.connected-app__note {
+  background-color: #f8f9fa;
+  border: none;
+  border-left: 4px solid #008397;
+  border-radius: 4px;
```

Comment:
Ideally we'd use a variable for the colour. `$color-3` is spree-blue in the old design, and the new teal in the new design.
Also, we can use a variable for border-radius. This helps ensure the left border is the same width as the radius.
```suggestion
  border-left: $border-radius solid $color-3;
  border-radius: $border-radius;
```
This will look a bit different on the old design, but I think should be ok.

Reply:
There could also be a variable for box-shadow, but I have a PR in progress for that, so let's just leave it hardcoded here for now.

Reply:
Ah, good to know. So `$color-3` is available in the old and new design? I guess, once the new design is released, we can do a simple search/replace.

Reply:
This is much better!
![Screenshot from 2023-12-15 16-22-15](https://github.com/openfoodfoundation/openfoodnetwork/assets/3524483/c61bfdd9-e77d-4fbe-8350-35edbcf998e2)
![Screenshot from 2023-12-15 16-22-32](https://github.com/openfoodfoundation/openfoodnetwork/assets/3524483/72f6ec60-1380-481c-afe3-71ff88be2cfb)